cpp/src/structure/create_graph_from_edgelist_impl.hpp

```diff
@@ -171,17 +163,16 @@ create_graph_from_edgelist_impl(raft::handle_t const& handle,
     renumber ? std::make_optional<rmm::device_uvector<vertex_t>>(0, handle.get_stream())
              : std::nullopt;
   std::optional<std::vector<vertex_t>> segment_offsets{std::nullopt};
+  renumber_meta_t<vertex_t, edge_t, multi_gpu> meta{};
   if (renumber) {
-    segment_offsets = std::vector<vertex_t>{};
-    std::tie(*renumber_map_labels, *segment_offsets) =
-      cugraph::renumber_edgelist<vertex_t, edge_t, multi_gpu>(
-        handle,
-        vertex_span ? std::optional<std::tuple<vertex_t const*, vertex_t>>{std::make_tuple(
-                        (*vertex_span).data(), static_cast<vertex_t>((*vertex_span).size()))}
-                    : std::nullopt,
-        store_transposed ? edgelist_cols.data() : edgelist_rows.data(),
-        store_transposed ? edgelist_rows.data() : edgelist_cols.data(),
-        static_cast<edge_t>(edgelist_rows.size()));
+    std::tie(*renumber_map_labels, meta) = cugraph::renumber_edgelist<vertex_t, edge_t, multi_gpu>(
```

Comment:
Same comment on `std::tie()` vs. structural bindings.

Reply:
This is not possible because renumber_map_labels & meta are defined outside the ```if (renumber) {}``` block.